cmake/features.cmake

```diff
@@ -28,6 +28,7 @@ ie_option (OS_FOLDER "create OS dedicated folder in output" OFF)
 # FIXME: ARM cross-compiler generates several "false positive" warnings regarding __builtin_memcpy buffer overflow
 ie_dependent_option (TREAT_WARNING_AS_ERROR "Treat build warnings as errors" ON "X86 OR X86_64" OFF)
 
+ie_option (ENABLE_INTEGRITYCHECK "force integrity check for windows binaries at load time (/INTEGRITYCHECK linker option)" OFF)
```

Comment:
Do we have a step in CI when `ENABLE_INTEGRITYCHECK` is ON?

Reply:
This option is not a default build yet. CI will update the signing steps in order to use this option.

Reply:
@ilyachur I have removed ENABLE_INTEGRITYCHECK from this PR